GETTING_STARTED.md

```diff
@@ -0,0 +1,875 @@
+question (see [Graph](docs/Reference/Graph/README.md) for details).
+
+Here is an example of a valid edge document:
+
+```
+    {
+        '_id': 'friends/001',
+        '_key': '001',
+        '_rev': '_Wm3dyle--_',
+        '_from': 'students/john',
+        '_to': 'students/jane',
+        'closeness': 9.5
+    }
+```
+
+---
+
+## 🔐 Auth - Connect to GDN
+
+The first step to start using GDN is establishing a connection to
+[play.paas.macrometa.io](https://play.paas.macrometa.io/). When this code executes, it
+initializes the server connection to your nearest region. You can access your
+Macrometa GDN account using several methods, such as
+[API keys](https://macrometa.com/docs/account-management/api-keys/),
+[User authentication](https://macrometa.com/docs/account-management/auth/user-auth),
+[Token-based authentication (JWT)](https://macrometa.com/docs/account-management/auth/jwts).
+
+### How to create and instance of jsc8?
+
+```js
+const jsc8 = require("jsc8");
+const client = new jsc8("https://play.paas.macrometa.io");
+```
+
+or to have fail-over support:
+
+```js
+const jsc8 = require("jsc8");
+const client = new jsc8([
+  "https://play1.paas.macrometa.io",
+  "https://play2.paas.macrometa.io",
+]);
+```
+
+This connection string actually represents the default value(
+`"https://play.paas.macrometa.io"` ), so if this is the value you want, you can omit
+url while invocation:
+
+```js
+const jsc8 = require("jsc8");
+const client = new jsc8();
+```
+
+If that’s still too verbose for you, you can invoke the driver directly using
+the require statement itself.
+
+```js
+const client = require("jsc8")();
+```
+
+The outcome of the three calls should be identical.
+
+### Auth with email and password
+
+To start working, you first have to log in. This gets the auth token and is
+added in each API call. Continuing from the previous snippet, the following
+statement will log you into the c8 using the provided credentials.
+
+```js
+async function login() {
+  return await client.login(email, password);
+}
+```
+
+Below line should print the jwt token. However, this is just for demonstrative
+purposes as this token will be added to further requests. Simple client.login
+should suffice for most cases.
+
+```js
+login().then(console.log);
+```
+
+To use the demotenant, we need to use client.useTenant as below:
+
+```js
+client.useTenant(tenant_name);
+```
+
+### Auth with token
+
+If you want to log in with token then you just need to pass the token wrapping
+in Object with your region URL with specific fabric.
+
+**Note**
+
+> Default value for fabric is `_system`
+
+```js
+const jsc8 = require("jsc8");
+client = new jsc8({
+  url: "https://play.paas.macrometa.io",
+  token: "XXXX",
+  fabricName: "_system",
+});
+```
+
+or to have fail-over support:
+
+```js
+const jsc8 = require("jsc8");
+client = new jsc8({
+  url: ["https://gdn1.macrometa.io", "https://gdn2.macrometa.io"],
```

Comment:
GDN should be changed to play federation.

Reply:
Yes Yes I will change it everywhere